tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -136,9 +136,9 @@ def stop_node_cb():
         await time_out_assert(5, wallet.get_confirmed_balance, initial_funds)
         await time_out_assert(5, wallet.get_unconfirmed_balance, initial_funds)
 
-        client = await WalletRpcClient.create(self_hostname, test_rpc_port, bt.root_path, config)
-        client_2 = await WalletRpcClient.create(self_hostname, test_rpc_port_2, bt.root_path, config)
-        client_node = await FullNodeRpcClient.create(self_hostname, test_rpc_port_node, bt.root_path, config)
+        client = await WalletRpcClient.create(hostname, test_rpc_port, bt.root_path, config)
+        client_2 = await WalletRpcClient.create(hostname, test_rpc_port_2, bt.root_path, config)
+        client_node = await FullNodeRpcClient.create(hostname, test_rpc_port_node, bt.root_path, config)
```

Comment:
Minor: seems like these changes could be reverted to just use the `self_hostname` fixture that's passed in. But I also see that `hostname` (which should be the same as `self_hostname`) is used above, so :shrug:

Reply:
Yeah, those two are equivalent